Snippets/Testing/Testing_6/Testing_6.csproj

```diff
@@ -7,5 +7,6 @@
     <PackageReference Include="NServiceBus.Gateway" Version="2.0.*" />
     <PackageReference Include="NServiceBus.Testing" Version="6.*" />
     <PackageReference Include="NUnit" Version="3.*" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.*" />
```

Comment:
seems like something that should be done on a diff pr

Reply:
Then I would have to install R# :) Trying to see if I could cope with out it :)